package.json

```diff
@@ -43,6 +43,7 @@
     "karma-sourcemap-loader": "0.3.5",
     "karma-webpack": "1.7.0",
     "lodash": "3.10.1",
+    "qunitjs": "^1.20.0",
```

Comment:
https://github.com/jsstyles/jss/blob/master/karma.conf.js#L11
You still have a dependency on it, without it installed:   (I am using npm3, so I don't get peerDependencies installed automatically)
Error: Cannot find module 'qunitjs'
    at Function.Module._resolveFilename (module.js:337:15)
    at Function.require.resolve (module.js:389:19)
    at initQUnit (/Users/kobrien/dev/c/jss/node_modules/karma-qunit/lib/index.j